tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,20 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+#@pytest.mark.skip
 def test_removing_not_found_returns_none():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
-
-    raise Exception("Complete this test according to comments below.")
+    #assert 
+    assert result == None
+    assert item not in vendor.inventory
+    assert len(vendor.inventory) == 3
```

Comment:
Great assertions, I would also suggest checking the inventory contents, since checks like the length being correct do not guarantee that the individual elements are still what we expect.

Reply:
Added these three assertions 
    assert "a" in vendor.inventory
    assert "b" in vendor.inventory
    assert "c" in vendor.inventory
Thank you.